serokell-style.md

```diff
@@ -109,12 +124,16 @@ data NetworkConfig = NetworkConfig
     }
 ```
 
+TODO: consider remove this rule? GHC has `-XOverloadedRecordFields` with
+[`HasField`](https://www.stackage.org/haddock/lts-10.4/base-4.10.1.0/GHC-Records.html#t:HasField)
+type class.
+
 **Library specific conventions**
 
 Add `F` suffix to custom formatters to avoid name conflicts:
 
 ```haskell
-nodeF :: Format r (NodeId -> r)
+nodeF :: NodeId -> Builder
```

Comment:
This change requires explanation.

Reply:
`fmt` library is better for formatting than `formatting` library. I think we could implicitly recommend better libraries. Though, every formatter in `fmt` is already have `F` suffix, so, probably there's no much sense in writing about this in style-guide because every reasonable person will add `F` suffix anyways.

Reply:
It doesn't really matter. This is just an example.